cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/PropertyUtils.java

```diff
@@ -0,0 +1,45 @@
+package cloud.foundry.cli.crosscutting.mapping;
+
+import cloud.foundry.cli.crosscutting.logging.Log;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Utils class for reading information from property files.
+ */
+public class PropertyUtils {
```

Comment:
As of yet, this class hardcodedly only handles *one* property file. I find the name slightly misleading. Maybe let's call it `VersionPropertiesUtils` (note the change to plural)?

Reply:
We have so far always used the singular in such cases. Why should I name this class in plural?
Why should I restrict the class to one use case by its naming? Now the other developers know that this class can be useful for any property file.

Reply:
The files are called `.properties`, and manage multiple properties regularly. Maybe `PropertiesFilesUtils` is more appropriate?

That said, the rename to `Version*` is more than appropriate, since the class solely implements utilities for this purpose. It's clearly not a generic class.

Reply:
Changed the name

Reply:
I did not know that you need a proper interface, an abstract base class for building a generic class.